### DISCUSSION TITLE
Usability nightmare in new REST API doc website

### DISCUSSION THREAD

**Discussion by stmllr** [community/community#24390] [Apps, API and Webhooks]
<p>Let’s talk about usability in <a href="https://docs.github.com/en/rest/reference" rel="nofollow noopener">https://docs.github.com/en/rest/reference</a> the REST API new home</p>
<p>The usability is IMHO a nightmare.</p>
<ol>
<li>API Response specs are an essential part of API doc and are hidden by default, need to click for folding out.</li>
<li>Browser search does not find anything in API response specs, unless I click and open the respective part.</li>
</ol>
<p>Steps to reproduce:</p>
<ol>
<li>Go to <a href="https://docs.github.com/en/rest/reference/users" rel="nofollow noopener">https://docs.github.com/en/rest/reference/users</a>
</li>
<li>Use browser search to find term “avatar_url”</li>
<li>No result</li>
<li>Click to open “default response” on any of the API sections</li>
<li>Retry search</li>
<li>Find expected results</li>
</ol>

**Comment by stmllr** [comment]
<p>The <a href="https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads" rel="nofollow noopener">Webhook events and payloads</a> serves as an example how to solve it.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/stmllr">@stmllr</a> welcome back to the community, thank you for taking the time to provide this feedback. I’ve passed it along to our docs team, and they have opened an issue to discuss the change. Thanks for your patience, as soon as I see an update on the issue I’ll post an update here :+1:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/stmllr">@stmllr</a>, I wanted to let you know we have a fix on the way! Will post back once is deployed.</p>

**Comment by stmllr** [comment]
<p>Looks like this has been fixed. Thank you!</p>
